### PR TITLE
Add indexing API and format links with English descriptions

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: best-practices/documentation/README.md [hunk 1]
```diff
@@ -19,13 +19,14 @@ OpenAPI Specifications may be expressed as either YAML or JSON. Thus far, we hav
...
-* https://sul-dlss.github.io/cocina-models/
-* https://sul-dlss.github.io/workflow-server-rails/ (in progress)
+* [SDR ("Cocina") Data Models](https://sul-dlss.github.io/cocina-models/)
+* [SDR Deposit API](https://sul-dlss.github.io/sdr-api/)
+* [SDR Identifier API](https://sul-dlss.github.io/suri-rails/)
+* [SDR Indexing API](https://sul-dlss.github.io/dor_indexing_app/)
+* [SDR Preservation API](https://sul-dlss.github.io/preservation_catalog/)
+* [SDR Repository ("DOR Services") API](https://sul-dlss.github.io/dor-services-app/)
+* [SDR Technical Metadata API](https://sul-dlss.github.io/technical-metadata-service/)
+* [SDR Workflow API](https://sul-dlss.github.io/workflow-server-rails/)
```
Comment (jmartin-sul) [inline best-practices/documentation/README.md:29]: i get a 404 on this one

Comment (mjgiarlo) [reply best-practices/documentation/README.md:29]: @jmartin-sul it's in progress. @aaron-collier is this slated for our current work cycle?

**Comment by mjgiarlo** [comment]
@jmartin-sul I can re-add the `(in progress)` qualifier, though my hope was that we'd be wrapping up that work in the current work cycle. maybe not? @aaron-collier

File: best-practices/documentation/README.md [hunk 2]
```diff
@@ -19,13 +19,14 @@ OpenAPI Specifications may be expressed as either YAML or JSON. Thus far, we hav
...
-* https://sul-dlss.github.io/cocina-models/
-* https://sul-dlss.github.io/workflow-server-rails/ (in progress)
+* [SDR ("Cocina") Data Models](https://sul-dlss.github.io/cocina-models/)
+* [SDR Deposit API](https://sul-dlss.github.io/sdr-api/)
+* [SDR Identifier API](https://sul-dlss.github.io/suri-rails/)
+* [SDR Indexing API](https://sul-dlss.github.io/dor_indexing_app/)
+* [SDR Preservation API](https://sul-dlss.github.io/preservation_catalog/)
+* [SDR Repository ("DOR Services") API](https://sul-dlss.github.io/dor-services-app/)
+* [SDR Technical Metadata API](https://sul-dlss.github.io/technical-metadata-service/)
+* [SDR Workflow API](https://sul-dlss.github.io/workflow-server-rails/) (**coming soon**!)
```
Comment (mjgiarlo) [inline best-practices/documentation/README.md:29]: @jmartin-sul Check it.

**Comment by jmartin-sul** [comment]
> @jmartin-sul I can re-add the `(in progress)` qualifier, though my hope was that we'd be wrapping up that work in the current work cycle. maybe not? @aaron-collier

ah, sorry!  with the removal of the qualifier, i figured the link would be available, and didn't realize the "in progress" meant it wasn't yet available.

anyway, do slightly prefer the re-addition, so thanks for putting the qualifier back.  merging...

**Pull request merged**